hazelcast/src/main/java/com/hazelcast/client/impl/protocol/template/ClientMessageTemplate.java

```diff
@@ -354,4 +354,16 @@ Object authenticationCustom(Data credentials, @Nullable String uuid, @Nullable S
     @Request(id = 18, retryable = false, response = ResponseMessageConst.VOID, event = {EventMessageConst.EVENT_PARTITIONS})
     @Since(value = "1.5")
     void addPartitionListener();
+
+    /**
+     * @param target  target address to create proxies on
+     * @param proxies proxies that will be created
+     *                If member is lite member replicated map will not be created and will be ignored silently.
+     *                Each entry's key is distributed object name.
+     *                Each entry's value is service name.
+     *                For possible service names see createProxy message.
+     */
+    @Since(value = "1.6")
+    @Request(id = 19, retryable = false, response = ResponseMessageConst.VOID)
+    void createProxies(List<Map.Entry<String, String>> proxies, Address target);
```

Comment:
how will it work if some of the proxy creations fail?

Reply:
The only reason to fail is replicated map trying to be created on a lite member. And it is silently ignored. Behaviour is caused by `PostJoinProxyOperation` since that operation is used. 
But if one fails, the ones that failed is just logged on the server side. But all will be attempted to be created.